src/Makefile.am

```diff
@@ -114,6 +114,7 @@ LIBZCASH_H = \
 .PHONY: FORCE collate-libsnark check-symbols check-security
 # bitcoin core #
 BITCOIN_CORE_H = \
+  addressindex.h \
```

Comment:
I only just discovered that these changes to this file are needed (apparently, although I haven't seen any ill-effects by not adding these lines). The header file names listed here are in alphabetical order.